### ISSUE TITLE
Update Email Templates

### ISSUE THREAD

**Issue by bg85** [auth0/auth0-deploy-cli#13]
We would also love to be able to update Email Templates

**Comment by joe-tillotson** [comment]
Agree, seems like email templates would be low hanging fruit relative to some of other other not-yet-supported items.

**Comment by adamtickner** [comment]
This would be good. I see it is already labelled as an enhancement for github deploys https://github.com/auth0-extensions/auth0-github-deploy/issues/25

**Comment by abbaspour** [comment]
working on this one. plz assign it to me.

**Comment by Sheepow** [comment]
@smarajitdasgupta There is a way to deploy and request email templates using the  [deprecated version of the management API](https://auth0.com/docs/api/management/v1#email-templates) . I recently tried it out to lookup for some of the template's status flags as they are not displayed by default in the dashboard.

**Comment by recursv** [comment]
There seems a way to deploy email templates using [v2](https://auth0.com/docs/api/management/v2#!/Email_Templates/patch_email_templates_by_templateName)
Can we not use this?

**Comment by Sheepow** [comment]
@recursv yes, this was recently added to v2, you can use those endpoints instead.

**Comment by stevezau** [comment]
This is now supported in v2. Please see the latest readme and example directory
